src/lib/utils/parser/config-validator.go

```diff
@@ -0,0 +1,106 @@
+package parser
+
+import (
+	"log"
+)
+
+// ValidateRoutesProp validates the `routes` property
+// in the configuration file.
+func ValidateRoutesProp(routes []Routes) {
+	if len(routes) == 0 {
+		log.Fatalf("`routes` property is missing.")
+	} else {
+		for i, route := range routes {
+			if route.Method == "" {
+				log.Fatalf("`method` property for route #%d is missing.", i+1)
+			}
+			if route.Route == "" {
+				log.Fatalf("`route` property for route #%d is missing.", i+1)
+			}
+			if route.URL == "" {
+				log.Fatalf("`url` property for route #%d is missing.", i+1)
+			}
+			if len(route.Header) != 0 {
+				for j, header := range route.Header {
+					if header.OfType == "" {
+						log.Fatalf("`type` property for header #%d of route `%s` is missing.", j+1, route.Route)
+					}
+					if header.Value == "" {
+						log.Fatalf("`value` property for header #%d of route `%s` is missing.", j+1, route.Route)
+					}
+				}
+			}
+			if len(route.Params) != 0 {
+				for j, param := range route.Params {
+					if param.Name == "" {
+						log.Fatalf("`name` property for param #%d of route `%s` is missing.", j+1, route.Route)
+					}
+					if param.Value == "" {
+						log.Fatalf("`value` property for param #%d of route `%s` is missing.", j+1, route.Route)
+					}
+				}
+			}
```

Comment:
Same goes for this. Please have a look and implement the same.

Reply:
same for `params`.